cpu/stm32_common/periph/rtt_lptim.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_stm32_common
+ * @{
+ *
+ * @file
+ * @brief       RTT implementation using LPTIM1
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "irq.h"
+#include "periph/rtt.h"
+
+/* this driver is only valid for STM CPUs that provide LPTIMERs */
+#ifdef LPTIM1
```

Comment:
Maybe changing this line to:
```c
#if (defined(LPTIM1) && defined(RTT_NUMOF) && RTT_NUMOF)
```
will fix the CI until the L0 boards are correctly configured.